### PR TITLE
[APPSRE-9762] add expiration for building rc images

### PR DESCRIPTION
**Pull request by BumbleFeng**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -39,7 +39,7 @@ push:
 	@$(CONTAINER_ENGINE) --config=$(DOCKER_CONF) push $(IMAGE_NAME):$(IMAGE_TAG)
 
 rc: git_version
-	@$(CONTAINER_ENGINE) build -t $(IMAGE_NAME):$(IMAGE_TAG)-rc -f dockerfiles/Dockerfile --target prod-image .
+	@$(CONTAINER_ENGINE) build -t $(IMAGE_NAME):$(IMAGE_TAG)-rc --build-arg quay_expiration=3d -f dockerfiles/Dockerfile --target prod-image .
```
Comment (hemslo) [inline Makefile:42]: `rc` looks like a temp tag, when are we using `make rc`?

**Review by hemslo — APPROVED** [review]
LGTM! Is there an issue linked? Maybe need to expire all existing `rc` images in quay too.

File: Makefile (hunk 1 above)
Comment (BumbleFeng) [reply Makefile:42]: rc image can be used for testing in app-interface pr-check, especially when the code change may take effect on multiple integrations.

**Comment by BumbleFeng** [comment]
I will clean all the rc images once this get merged.

**Pull request merged**